src/inc/eventtracebase.h

```diff
@@ -216,11 +300,21 @@ extern "C" {
 #endif //!DONOT_DEFINE_ETW_CALLBACK && !DACCESS_COMPILE
 
 #include <clretwallmain.h>
+
+#elif defined(PLATFORM_UNIX)
+
+#include "clrallevents.h"
```

Comment:
Actually this header is for every platform except windows and is not specific to LTTng, Also it has nothing to do with tracepoints,it is a place holder for all the FireEtw* and EventEnabled* functions. Hence i am leaving it as it is